model-optimizer/mo/utils/broadcasting.py

```diff
@@ -134,7 +147,7 @@ def explicit_broadcasting(input_value: np.array, target_shape: np.array, axes_ma
     :return: broadcasted value
     """
     res_shape, normalized_axes_mapping = explicit_shape_broadcasting(input_value.shape, target_shape, axes_mapping)
+    #TODO: Function 'expand_dims' should be replaced with 'numpy.expand_dims' if numpy version will be >=18.x in requirements.
     expand_dim_axis = set(np.arange(len(target_shape))) - set(normalized_axes_mapping)
-
-    input_expanded = np.expand_dims(input_value.copy(), axis=list(expand_dim_axis))
+    input_expanded = expand_dims(input_value.copy(), axis=list(expand_dim_axis))
```

Comment:
I propose to do something like this


```suggestion
input_expanded = input_value.copy()
if numpy_version < 18.xx:
   for axis in expand_dim_axis:
       input_expanded = np.expand_dims(input_expanded, axis=axis)
else:
    input_expanded = np.expand_dims(input_expanded, axis=expand_dim_axis)

```
in the later case we will avoid code duplication.

Reply:
No need to have a check for a numpy version. Just implement a `for` loop regardless of the numpy version installed.

Reply:
fixed